RobotCode/snobot2018/src/org/snobot/elevator/SnobotElevator.java

```diff
@@ -48,30 +60,41 @@ private void caluculateHeight()
     {
 
         double encoderPosition = mElevatorEncoder.getDistance();
-        mHeight = encoderPosition;
+        mActualHeight = encoderPosition;
     }
 
 
     @Override
     public void control()
     {
-        setMotorSpeed(mElevatorSpeed);
 
+        if (mGotoHeight)
```

Comment:
Can the user go to a height?  We will likely have a position for switch, scale low, scale neutral, scale high that they would want to use in teleop